packages/cli/src/hl7.ts

```diff
@@ -0,0 +1,59 @@
+import { formatHl7DateTime, Hl7Message } from '@medplum/core';
+import { Hl7Client, Hl7Server } from '@medplum/hl7';
+import { Command } from 'commander';
+import { readFileSync } from 'fs';
+import { createMedplumCommand } from './util/command';
+
+const send = createMedplumCommand('send')
+  .description('Send an HL7 v2 message via MLLP')
+  .argument('<host>', 'The destination host name or IP address')
+  .argument('<port>', 'The destination port number')
+  .argument('[body]', 'Optional HL7 message body')
+  .option('--generate', 'Generate a sample HL7 message')
+  .option('--file <file>', 'Read the HL7 message from a file')
+  .action(async (host, port, body, options) => {
+    if (options.generate) {
+      body = generateSampleHl7Message();
+    } else if (options.file) {
+      body = readFileSync(options.file, 'utf8');
+    }
+
+    if (!body) {
+      throw new Error('Missing HL7 message body');
+    }
+
+    const client = new Hl7Client({
+      host,
+      port: parseInt(port, 10),
+    });
+
+    const response = await client.sendAndWait(Hl7Message.parse(body));
+    console.log(response.toString().replaceAll('\r', '\n'));
+    client.close();
```

Comment:
Does this need to be in a `try/finally` or similar to ensure the client is closed?